WISE-Community/WISE-Client
# feat(Match): Update grading view to match student view

## Changes
Grading view now matches what student sees for Match component in terms of styles, layout, and feedback.

Closes #401.

## Test
- Make sure Match components work as before as a student.
- Verify that grading view matches what student sees.
- Test with Match components that have "ordered=true" set to verify that position feedback works as well.
